corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -143,15 +160,11 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search Properties" %}</h4>
         <table class="table table-condensed">
           <thead data-bind="visible: searchProperties().length > 0">
           <tr>
-            <th class="col-sm-3">{% trans "Case Property" %}</th>
-            <th class="col-sm-4">{% trans "Display Text" %}</th>
-            {% if js_options.default_value_expression_enabled %}
-              <th class="col-sm-3">{% trans "Default Value Expression" %}</th>
-            {% endif %}
-            {% if js_options.search_prompt_appearance_enabled %}
-              <th class="col-sm-2">{% trans "Format" %}</th>
+            <th class="col-sm-4">{% trans "Case Property" %}</th>
+            <th class="col-sm-5">{% trans "Display Text" %}</th>
+            {% if js_options.search_prompt_appearance_enabled or js_options.default_value_expression_enabled%}
+              <th class="col-sm-4">{% trans "More Options" %}</th>
```

Comment:
Optional: the button doesn't take up that much space, you could give some space back to the property & name text boxes.

Reply:
Sure.